catalog/controller/extension/payment/razorpay.php

```diff
@@ -350,4 +391,105 @@ protected function getApiIntance()
         return new Api($this->config->get('payment_razorpay_key_id'), $this->config->get('payment_razorpay_key_secret'));
     }
 
+    protected function autoCreateWebhook()
+    {
+        $this->webhookSecret = $this->config->get('payment_razorpay_webhook_secret');
+
+        if(empty($this->webhookSecret) === true)
+        {
+            $this->webhookSecret = bin2hex(openssl_random_pseudo_bytes(4));
```

Comment:
Can we increase length of secret here. Also it will always use lowercase letters and numbers. We must include uppercase letters as well.

Reply:
@ramth05 added createWebhookSecret function